apps/src/studio/levels.js

```diff
@@ -2703,10 +2711,14 @@ levels.js_hoc2015_chain_characters = {
     {
       'id': 'playlab:js_hoc2015_chain_characters:calloutPlaceTwoWhenTauntaun',
       'element_id': '.droplet-gutter-line:nth-of-type(9)',
+      'hide_target_selector': '.droplet-drag-cover',
       'qtip_config': {
         'content' : {
           'text': msg.calloutPlaceTwoWhenTauntaun(),
         },
+        'hide': {
+          'event': 'mouseup touchend',
```

Comment:
`hide_target_selector` is sufficient if you want to make it hide when you click on something other than the element you attached to.
`hide.event` is also needed if you want to make it hide on an event other than `click` - in this case, because we want to look for a drop, we give it `mouseup` and `touchend`